inbm/dispatcher-agent/dispatcher/schedule/schedules.py

```diff
@@ -6,25 +6,30 @@
 """
 from datetime import datetime
 from dataclasses import dataclass, field
-from typing import Optional
+from typing import Optional, List
 
-@dataclass(init=True)
-class SingleSchedule:
-    """ Represents a SingleSchedule object from the Scheduled Manifest """
+@dataclass
+class Schedule:
+    """ Represents a Base class for schedule objects """
     request_id: str
+    id: Optional[int] = field(default=None)    
+
+
+@dataclass
+class SingleSchedule(Schedule):
+    """ Represents a SingleSchedule object from the Scheduled Manifest """ 
     start_time: Optional[datetime] = field(default=None)
     end_time: Optional[datetime] = field(default=None)
-    manifests: list[str] = field(default_factory=list)
+    manifests: List[str] = field(default_factory=list)
```

Comment:
list[str] is preferred as it doesn't require an import

Reply:
The AI asked me to change it.  I believe this came up on another PR.  I don't know why it wants that for that, but it insists.